prow/cmd/tot/main.go

```diff
@@ -35,12 +37,14 @@ var (
 	port        = flag.Int("port", 8888, "port to listen on")
 	logJson     = flag.Bool("log-json", false, "output log in JSON format")
 	storagePath = flag.String("storage", "tot.json", "where to store the results")
+	fallbackURI = flag.String("fallback", "", "HTTP endpoint to GET for missing build numbers -- template with %s")
```

Comment:
It is correct as is, but confusing and unidiomatic. I wouldn't even bother allowing templating, just hard-code the gubernator link. We're going to add this sort of thing to a config file in the future, and I'd rather avoid ad-hoc templates in arguments.